src/saga/adaptors/context/myproxy.py

```diff
@@ -135,7 +148,7 @@ def _initialize (self, session) :
             cmd += " --proxy_lifetime %s"  %  api.life_time
 
         # store the proxy in a private location
-        proxy_store    = "%s/.saga/proxies/"   %  os.environ['HOME']
+        proxy_store    = "%s"   %  self.base_work_dir
```

Comment:
The default location now changed from `~/.saga/proxies/` to `~/.saga/adaptors/proxies/`.  I don't mind much, but wanted to check if this was intentional?